### PR TITLE
Update fields.md

### PR DESCRIPTION
**Pull request by Tastaturberuf**
https://github.com/contao/contao/commit/9a8e91681a4d1acaf45e3ab48584f437be7a9078

### CODE REVIEW

**Review by fritzmg — APPROVED** [review]

**Comment by fritzmg** [comment]
Thank you @Tastaturberuf :)

File: docs/dev/reference/dca/fields.md [hunk 1]
```diff
@@ -185,6 +185,7 @@ Each field can be validated against a regular expression.
 | unique             | true/false (`bool`)           | If true the field value cannot be saved if it exists already.                                                                                                            |
 | uploadFolder       | Path (`string`)                  | The target path for file uploads of the `upload` widget.                                                                                                              |
 | useRawRequestData  | true/false (`bool`)           | If true the raw request data from the Symfony request is used. **Warning:** input filtering is bypassed! Make sure the data is never output anywhere in the back end unescaped which it would if you added the field to a regular back end list view for example. |
+| versionize         | true/false (`bool`)           | If false skip this field in the versioning. Default `true`.
```
Comment (fiedsch) [inline docs/dev/reference/dca/fields.md:188]: Maybe
If `false`, skip this field in versioning. Default `true`.

Comment (fiedsch) [reply docs/dev/reference/dca/fields.md:188]: Sorry, too late ... :-(

Comment (Tastaturberuf) [reply docs/dev/reference/dca/fields.md:188]: I think that's good, but then we would have to adjust the entire table.

Comment (Tastaturberuf) [reply docs/dev/reference/dca/fields.md:188]: Ah, i see you have removed the `the` also. only see you have write false as `false`.

Comment (fiedsch) [reply docs/dev/reference/dca/fields.md:188]: The "the" sounds strange to me, but I don't know if my suggestion is proper english. 
Or maybe "when versioning".

Comment (Tastaturberuf) [reply docs/dev/reference/dca/fields.md:188]: i dont know proper english ¯\_(ツ)_/¯

**Pull request merged**